src/Kdyby/Doctrine/Entities/IdentifiedEntity.php

```diff
@@ -26,9 +25,11 @@
  *
  * @deprecated Use Kdyby\Doctrine\Entities\Attributes\Identifier trait instead.
  */
-abstract class IdentifiedEntity extends BaseEntity
+abstract class IdentifiedEntity
 {
 
+	use Kdyby\DoctrineMagicAccessors\MagicAccessors;
```

Comment:
this is technically a BC Break, but I do not care much, since this will go to 4.0